### ISSUE TITLE
Aerosol layer height on wiki

### ISSUE THREAD

**Issue by simonrp84** [ORAC-CC/orac#16]
Is there any reason we have this page: https://github.com/ORAC-CC/orac/wiki/Definitions
Linked to from the ORAC wiki homepage? It seems completley out of context. Can we merge with something more relevant?

**Comment by caropoulsen** [comment]
There was a reason but I cannot remember what it was. I quite liked having definitions for retrieved variables, some are not easy to understand for the non expert, on the website but would be better if we had the full set.


I think Don created that page.

Caroline



________________________________

**Comment by adamcpovey** [comment]
Don created the page back before my uncertainty paper or an equivalent he wanted Greg to write about clouds. The idea was that fairly fundamental terms, such as aerosol optical depth and cloud top height, are used in subtly different ways across the field. He thought we should (a) be clearly what exactly **we** mean by all our variable names and (b) try to browbeat others into agreeing with us.

Maybe we make a third track to the Wiki: Software installation and use; Product descriptions; General description of aerosol/cloud retrievals

**Comment by simonrp84** [comment]
Good idea to separate this out into another part of the wiki - telling people what ORAC is and telling people what we think definitions should be are best separated. :-)
